djangoapp/app/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+import django_tables2 as tables
```

Comment:
Similar to `cripsy_forms` I'd avoid using a third-party library here.

This is also a bit misleading; I had to Google what `django_tables2` is, assuming it was a DB-related library because it's used in `models.py`, but it looks like it's actually a HTML helper?

Reply:
yes, it's just a tables decorator, useful to make output responsive